.github/workflows/update-pull-request.yml

```diff
@@ -4,6 +4,15 @@ on:
   issue_comment:
     types:
       - created
+  workflow_call:
+    secrets:
+      PULL_REQUEST_UPDATE_TOKEN:
```

Comment:
Where is this used?

Reply:
Mainly as token for `actions/checkout@v4`:

https://github.com/MetaMask/snaps/blob/1057c837033fca3e1c5e610fddd6a36a41ed1313/.github/workflows/update-pull-request.yml#L196-L198